dashboard/src/actions/charts.test.tsx

```diff
@@ -62,33 +81,6 @@ describe("fetchCharts", () => {
     );
   });
 
-  it("fetches charts with query", async () => {
```

Comment:
why have you deleted this test? (even if the query is not sent in the action it's useful to ensure it's being used)

Reply:
The action no longer has the "query" as part of the payload. Why would we have to still have this test for a non-existent parameter? 

```javascript 
export const requestCharts = createAction("REQUEST_CHARTS", resolve => {
  return (page?: number) => resolve(page);
});
```

So now the tests are testing there `page` parameter, so new test cases have been added for:

```
fetches charts from a repo (first page)
fetches charts from a repo (middle page)
fetches charts from a repo (last page)
fetches charts from a repo (already processed page)
fetches charts from a repo (not-yet-requested page)
fetches charts from a repo (already-requested page)
```